res/text.zh-TW.txt

```diff
@@ -0,0 +1,21 @@
+DungeonRush
+作者 Rapiz
+PLACEHOLDER
+PLACEHOLDER
+玩家1
```

Comment:
Please use UTF-8 encoding

Reply:
it's UTF-8 encoding
use UTF-8 win BOM?

Reply:
Oh. It looks strange in my browser so I thought it's GBK. Maybe it's my browser's fault. Fine.